src/views/account1/account1.tsx

```diff
@@ -0,0 +1,445 @@
+import React, { useState, useRef, useEffect } from 'react';
+import Footer from '@/components/global/footer/Footer';
+import Navbar from '@/components/global/navbar/Navbar';
+import circle from '../../../public/images/circle.png';
+import Rcircle from '../../../public/images/r-circle.png';
+import copy from '../../../public/images/content-copy.svg';
+import eye from '../../../public/images/eye.png';
+import plus from '../../../public/images/plus.png';
+import bell from '../../../public/images/bell.png';
+import shortBell from '../../../public/images/short-bell.png';
+import dot from '../../../public/images/dot.png';
+import pen from '../../../public/images/pen.png';
+import api from '../../../public/images/api.png';
+import arrow from '../../../public/images/uparrow.png';
+import refresh from '../../../public/images/refresh.png';
+import trash from '../../../public/images/delete.png';
+import Image from 'next/image';
+import ApexLineChart from '../../../src/views/account1/apexchart';
+
+const Account1 = () => {
+    const [isDropdownOpen, setDropdownOpen] = useState(false);
+
+    const dropdownRef:any = useRef(null);
+
+    const toggleDropdown = () => {
+        setDropdownOpen(!isDropdownOpen);
+    };
+
+    const handleClickOutside = (event: any) => {
+        if (dropdownRef.current && (!dropdownRef.current.contains((event.target) as any))) {
+            setDropdownOpen(false);
+        }
+    };
+
+    useEffect(() => {
+        document.addEventListener('mousedown', handleClickOutside);
+        return () => {
+            document.removeEventListener('mousedown', handleClickOutside);
+        };
+    }, []);
+
+    return (
+        <>
+            <div className="hello">
+                <Navbar searchbar />
+                <div className="container py-3 px-[16px] w-full">
+                    <p style={{ fontSize: '34px', fontWeight: 'bolder' }}>API Keys</p>
+                    <div style={{ display: 'flex' }} className="grid gap-2 mt-6 font-bold text-center alignItem-center">
+                        <div
+                            className="rounded border flex align-items-center justify-center gap-2 bg-black text-white text-center"
+                            style={{ fontSize: '14px', padding: '10px 30px' }}
+                        >
+                            <Image src={plus} alt="" className="h-3 w-3" style={{ marginTop: '4px' }} />
+                            <p>CREATE NEW KEY</p>
+                        </div>
+                        <div
+                            className="rounded border flex align-items-center justify-center text-center gap-2"
+                            style={{ fontSize: '14px', padding: '10px 30px' }}
+                        >
+                            <Image src={api} alt="" className="h-5 w-6" style={{ marginTop: '2px' }} />
```

Comment:
please check the css of all the texts in the page. They look different from the designs